src/os/bluestore/NVMEDevice.cc

```diff
@@ -865,6 +865,7 @@ int NVMEDevice::open(const string& p)
   driver->register_device(this);
   block_size = driver->get_block_size();
   size = driver->get_size();
+  name = serial_number;
```

Comment:
not, like, path?

and so I'm confused by this; looks like spdk sets it to the result of spdk_pci_device_get_device_name, but
https://github.com/spdk/spdk/commit/dca887b7d8e875262d6c523450a4fc1576835f65.  Granted, it's only used in messages apparently, but, it seems like it ought to be useful as an identifier for the device to humans.  What's the intent?